### PR TITLE
'Dogecoin & Coin Burning' Article

### PR DESCRIPTION
**Pull request by RichieJohn**
Simplez. ;P

### CODE REVIEW

File: content/dogepedia/faq/dogecoin-and-coin-burning.md [hunk 1]
```diff
@@ -5,4 +5,6 @@ date = "2021-10-22"
   name = "Dogecoin"
 +++
 
-Dogecoin is not a speculative asset, and it does not need coin burning mechanisms new tokens and other chains have attempted to implement to artificially inflate their prices.
+Dogecoin is not a speculative asset, and it does not need coin burning marketing mechanisms that new tokens and other chains have attempted to implement to artificially inflate their prices.
```
Comment (AbcSxyZ) [inline content/dogepedia/faq/dogecoin-and-coin-burning.md:8]: > Dogecoin is not a speculative asset

I'm not sure about it :rofl:

Comment (RichieJohn) [reply content/dogepedia/faq/dogecoin-and-coin-burning.md:8]: Will update now! 😂

**Review by RichieJohn — COMMENTED** [review]
Needs a review and/or approval for merge to main @AbcSxyZ @MishaBoar

**Review by Mishaboar2 — APPROVED** [review]
Simplez.

**Review by xanimo — APPROVED** [review]
ACK

**Pull request merged**